plugins/synaptics-rmi/fu-synaptics-rmi-firmware.c

```diff
@@ -529,7 +594,7 @@ fu_synaptics_rmi_firmware_parse(FuFirmware *firmware,
 				    error))
 		return FALSE;
 	fu_firmware_set_size(firmware, firmware_size);
-
+	g_debug("bootloader version is %d \n", self->bootloader_version);
```

Comment:
why this? isn't it in the to_string() output as well?

Reply:
Removed.